src/Redis.php

```diff
@@ -3537,6 +3537,52 @@ public function bzPopMin($key1, $key2, $timeout)
     {
     }
 
+    /**
+     * Can pop the highest scoring members from one ZSET.
+     *
+     * @param string $key
+     * @param int $count
+     *
+     * @return array Either an array with the key member and score of the highest element or an empty array
+     * if there is no element to pop.
+     *
+     * @since >= 5.0
+     * @link https://redis.io/commands/zpopmax
+     * @example
+     * <pre>
+     * // Pop the *lowest* scoring member from set `zs1`.
+     * $redis->zPopMax('zs1');
+     * // Pop the *lowest* 3 scoring member from set `zs1`.
+     * $redis->zPopMax('zs1', 3);
+     * </pre>
+     */
+    public function zPopMax($key, $count = 1)
```

Comment:
`zPopMax(string $key, int $count = 1)`

Reply:
No there are no typehints in the whole extension. If you extend and overwrite it will error that your function is incompatible to your parent class if you add typehints to this function.

Reply:
ok, thank you